functional-test/src/test/groovy/org/rundeck/util/common/execution/ExecutionUtils.groovy

```diff
@@ -40,6 +40,18 @@ class ExecutionUtils {
             }
         }
 
+        /**
+         * Returns a boolean that verifies if the executions where executed sequentially,
+         * meaning the first execution date-ended is lesser than the second's start-date.
+         *
+         * @param executions is the list of executions to verify sorted from first executed to last executed.
+         * @return Boolean
+         */
+        static final haveExecutedSequentially(List<Execution> executions) {
```

Comment:
The `org.rundeck.util.api.responses.execution.Execution` type does not provide `date-started` or `date-ended` properties. You're relying on the "hidden" jackson deserialization artifacts that are not guaranteed.